articles/quickstarts/install-python.md

```diff
@@ -40,53 +62,44 @@ Install the QDK to develop Python host programs to call Q# operations.
     > ```
     > where `/path/to/dotnet-iqsharp` should be replaced by the absolute path to the `dotnet-iqsharp` tool in your file system.
     > Typically this will be under `.dotnet/tools` in your user profile folder.
-  
-1. While you can use Q# with Python in any IDE, we highly recommend using Visual Studio Code (VS Code) IDE for your Q# + Python applications. By using Visual Studio Code and the QDK Visual Studio Code extension you gain access to richer functionality.
+    
+***
 
-    - Install [VS Code](https://code.visualstudio.com/download) (Windows, Linux and Mac)
-    - Install the [QDK extension for VS Code](https://marketplace.visualstudio.com/items?itemName=quantum.quantum-devkit-vscode).
+That's it! You now have both the `qsharp` Python package and the IQ# kernel for Jupyter, which provides the core functionality for compiling and executing Q# operations from Python and allows you to use Q# Jupyter Notebooks.
 
-1. Verify the installation by creating a `Hello World` application
+## Write your first Q# program
 
-    - Create a minimal Q# operation, by creating a file called `Operation.qs`, and adding the following code to it:
+While you can use Q# with Python in any IDE, we highly recommend using Visual Studio Code (VS Code) IDE for your Q# + Python applications. By using Visual Studio Code and the QDK Visual Studio Code extension you gain access to richer functionality.
```

Comment:
```suggestion
While you can use Q# with Python in any IDE, we highly recommend using Visual Studio Code (VS Code) IDE for your Q# + Python applications. With the QDK Visual Studio Code extension you gain access to richer functionality such as warnings, syntax highlighting, project templates, and more.

If you would like to use VS Code:
```

Reply:
Ok, I believe I've merged your changes in correctly, but please double-check!